Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM golang:1.16 as builder
+FROM golang:1.18 as builder
```

Comment:
I see that the go version in `go.mod` is still 1.16, shouldn't we use the same version in both places?